htslib/sam.h

```diff
@@ -1110,21 +1110,21 @@ int bam_set_qname(bam1_t *b, const char *qname);
                        can be NULL
  @param  a_cigar [out]  address of the destination uint32_t buffer
  @param  a_mem   [in/out]  address of the allocated number of buffer elements
- @return         number of processed CIGAR operators; 0 if error
+ @return         number of processed CIGAR operators; -1 on error
  */
 HTSLIB_EXPORT
-size_t sam_parse_cigar(const char *in, char **end, uint32_t **a_cigar, uint32_t *a_mem);
+ssize_t sam_parse_cigar(const char *in, char **end, uint32_t **a_cigar, uint32_t *a_mem);
```

Comment:
By luck _sam.h_ currently gets it via _kstring.h_, but in case that changes: need to add `#include <sys/types.h>` at the top of _sam.h_ to ensure `ssize_t` is declared.

Reply:
Thanks for the catch.